wagtail_streamfield_migration_toolkit/operations.py

```diff
@@ -151,6 +174,12 @@ def map_temp_blocks_to_list_items(self):
             new_temp_blocks.append({**block, "type": "item"})
         self.temp_blocks = new_temp_blocks
 
+    @property
+    def operation_name_fragment(self):
+        return "convert_{}_to_ListBlock_{}".format(
```

Comment:
We should snake-case the block names, so we don't generate mixed-case filenames.

Maybe we should drop "convert" from all of these - "stream_block_to_list_block" is just as descriptive as "convert_stream_block_to_list_block" in this context.